deploy/utils.sh

```diff
@@ -388,3 +389,51 @@ verify_secret_value_in_pod() {
   $cli_with_timeout "exec -n $APP_NAMESPACE_NAME ${pod_name} -- \
     printenv | grep $environment_variable_name | cut -d '=' -f 2- | grep $expected_value"
 }
+
+get_apps_logs_container() {
+    echo "Get app logs"
+    set_namespace "$APP_NAMESPACE_NAME"
+    helm=$(helm ls -aq)
+    $cli_without_timeout get pods
```

Comment:
does the namespace matter here?

Reply:
`$cli_without_timeout get pods --namespace $APP_NAMESPACE_NAME`

Reply:
No, since i'm inside the namespace `set_namespace "$APP_NAMESPACE_NAME"`

Reply:
I would still add it to be explicit but whatever you choose

Reply:
i will leave it without it, this is the reason i set to this namespace, to avoid from all the command to include the namespace.